tools/make/golang.mk

```diff
@@ -65,8 +65,8 @@ go.format:  go.format.verify
 build: ## Build envoy-gateway for host platform. See Option PLATFORM and BINS.
 	@$(MAKE) go.build
 
-.PHONY: build.multiarch
-build.multiarch: ## Build envoy-gateway for multiple platforms. See Option PLATFORMS and IMAGES.
+.PHONY: build-multiarch
+build-multiarch: ## Build envoy-gateway for multiple platforms. See Option PLATFORMS and IMAGES.
```

Comment:
Any good reasons to change the `.` to `-`?

Reply:
it doesn't show up in `make help`, worth debugging why :)

Reply:
Got it, thanks @arkodg and @daixiang0